content/en/docs/reference/setup-tools/kubeadm/kubeadm-alpha.md

```diff
@@ -18,15 +18,33 @@ You can renew all Kubernetes certificates using the `all` subcommand or renew th
 {{< tabs name="tab-certs-renew" >}}
 {{< tab name="renew" include="generated/kubeadm_alpha_certs_renew.md" />}}
 {{< tab name="all" include="generated/kubeadm_alpha_certs_renew_all.md" />}}
+{{< tab name="admin.conf" include="generated/kubeadm_alpha_certs_renew_admin.conf.md" />}}
 {{< tab name="apiserver-etcd-client" include="generated/kubeadm_alpha_certs_renew_apiserver-etcd-client.md" />}}
 {{< tab name="apiserver-kubelet-client" include="generated/kubeadm_alpha_certs_renew_apiserver-kubelet-client.md" />}}
 {{< tab name="apiserver" include="generated/kubeadm_alpha_certs_renew_apiserver.md" />}}
+{{< tab name="controller-manager.conf" include="generated/kubeadm_alpha_certs_renew_controller-manager.conf.md" />}}
 {{< tab name="etcd-healthcheck-client" include="generated/kubeadm_alpha_certs_renew_etcd-healthcheck-client.md" />}}
 {{< tab name="etcd-peer" include="generated/kubeadm_alpha_certs_renew_etcd-peer.md" />}}
 {{< tab name="etcd-server" include="generated/kubeadm_alpha_certs_renew_etcd-server.md" />}}
 {{< tab name="front-proxy-client" include="generated/kubeadm_alpha_certs_renew_front-proxy-client.md" />}}
+{{< tab name="scheduler.conf" include="generated/kubeadm_alpha_certs_renew_scheduler.conf.md" />}}
 {{< /tabs >}}
 
+## kubeadm alpha certs certificate-key {#cmd-certs-certificate-key}
+
+This command can be used to generate a new control-plane ceritificate key.
+
+{{< tabs name="tab-certs-certificate-key" >}}
+{{< tab name="certificate-key" include="generated/kubeadm_alpha_certs_certificate-key.md" />}}
+{{< /tabs >}}
+
+## kubeadm alpha certs check-expiration {#cmd-certs-check-expiration}
+
+This command checks expiration for the certificates in the local PKI managed by kubeadm.
```

Comment:
If possible add a link to the certificate doc; 
if not a blast, it would be great to do the same for alpha certs renew and upgrade apply...